src/main/java/org/broadinstitute/hellbender/engine/spark/AddContextDataToReadSpark.java

```diff
@@ -31,78 +32,20 @@
 public class AddContextDataToReadSpark {
     public static JavaPairRDD<GATKRead, ReadContextData> add(
             final JavaRDD<GATKRead> reads, final ReferenceDataflowSource referenceDataflowSource,
-            final JavaRDD<Variant> variants) {
-
-        // This transform can currently only handle mapped reads
+            final JavaRDD<Variant> variants, final JoinStrategy joinStrategy) {
+        // TODO: this static method should not be filtering the unmapped reads.  To be addressed in another issue.
```

Comment:
The way to address this is to add support in `AddContextDataToReadSpark` for unmapped reads -- could you create a ticket for this task?